### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by cordwelt**

### CODE REVIEW

File: .github/workflows/dev-grpc-services-pypi-publish.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+name: Dev build and publish Python distributions to PyPI. New gRPC generator
```
Comment (Nikita-Smirnov-Exactpro) [inline .github/workflows/dev-grpc-services-pypi-publish.yml]:
```suggestion
name: Dev build and publish Python distributions to PyPI.
```

File: .github/workflows/grpc-services-pypi-publish.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+name: Build and publish Python distributions to PyPI. New gRPC generator
```
Comment (Nikita-Smirnov-Exactpro) [inline .github/workflows/grpc-services-pypi-publish.yml]:
```suggestion
name: Build and publish Python distributions to PyPI.

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+            artifact = "com.exactpro.th2:grpc-service-generator:${serviceGeneratorVersion}:all@jar"
+        }
+    }
+
+    generateProtoTasks.generatedFilesBaseDir = genDir
+
+    generateProtoTasks {
+        all()*.plugins {
+            grpc {}
+            rxgrpc {}
```
Comment (Nikita-Smirnov-Exactpro) [inline build.gradle]: Maybe redundant

File: src/main/proto/th2_grpc_conn/conn.proto [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+package conn;
```
Comment (Nikita-Smirnov-Exactpro) [inline src/main/proto/th2_grpc_conn/conn.proto]: I'm not sure about this line. Maybe we will have a problem in python

File: build.gradle (hunk 3 above)
Comment (cordwelt) [reply build.gradle]: How can I check if it's redundant?

Comment (Nikita-Smirnov-Exactpro) [reply build.gradle]: We use rxjava3 in th2-conn instead of v2

Comment (cordwelt) [reply build.gradle]: ```We use rxjava3 in th2-conn instead of v2```

I don't get it. Could you please elaborate on that?

Comment (Nikita-Smirnov-Exactpro) [reply build.gradle]: The rxgrpc plugin (implementation 'io.reactivex.rxjava2:rxjava:2.2.19') generate stubs for using in a rxjava style. jar with the current grpc classes will have dependencies on rxjava2. 
Neither of our components uses rxjava2 and the additional dependency will be redundant in our case

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

**Pull request merged**